src/tests/tests_cinder_lvm.py

```diff
@@ -82,7 +85,7 @@ def test_cinder_config(self):
             model_name=self.model_name,
             timeout=10)
 
-    def _tst_create_volume(self):
+    def _create_volume(self):
```

Comment:
this is changed here, but it should be changed in the PR for zaza-openstack-tests and this whole class dropped from this PR please.